internal/kafka/test/integration/kafka_create_test.go

```diff
@@ -0,0 +1,528 @@
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/kafkas/types"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/presenters"
+	kafkatest "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test/common"
+	mockkafkas "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test/mocks/kafkas"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/test/mocks/kasfleetshardsync"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/metrics"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/test/mocks"
+
+	"github.com/onsi/gomega"
+)
+
+// Test Kafka creation with manual scaling enabled
+func TestKafkaCreate_ManualScaling(t *testing.T) {
+	g := gomega.NewWithT(t)
+
+	ocmServer := mocks.NewMockConfigurableServerBuilder().Build()
+	defer ocmServer.Close()
+
+	var enableManualScaling bool
+	clusterList := config.ClusterList{}
+	h, client, teardown := kafkatest.NewKafkaHelperWithHooks(t, ocmServer, func(d *config.DataplaneClusterConfig) {
+		if enableManualScaling {
+			// Enable manual scaling and register data plane cluster to cluster config
+			d.DataPlaneClusterScalingType = config.ManualScaling
+			d.ClusterConfig = config.NewClusterConfig(clusterList)
+		}
+	})
+	defer teardown()
+
+	// run mock fleetshard sync
+	mockKasFleetshardSyncBuilder := kasfleetshardsync.NewMockKasFleetshardSyncBuilder(h, t)
+	mockKasFleetshardSync := mockKasFleetshardSyncBuilder.Build()
+	mockKasFleetshardSync.Start()
+	defer mockKasFleetshardSync.Stop()
+
+	// set up data plane cluster
+	clusterID, getClusterErr := common.GetOSDClusterIDAndWaitForStatus(h, t, api.ClusterReady)
+	g.Expect(getClusterErr).ToNot(gomega.HaveOccurred(), "failed to get dataplane cluster details: %v", getClusterErr)
+	g.Expect(clusterID).ToNot(gomega.BeEmpty(), "no dataplane cluster found")
+
+	// reload services with manual scaling enabled
+	h.Env.Stop()
+
+	// register existing cluster to the data plane cluster config
+	db := h.DBFactory().New()
+	cluster := api.Cluster{
+		ClusterID: clusterID,
+	}
+	err := db.First(&cluster).Error
+	g.Expect(err).ToNot(gomega.HaveOccurred(), "failed to get data plane cluster from database: %v", err)
+
+	clusterList = append(clusterList, config.ManualCluster{
+		Name:                  "test-cluster",
+		ClusterId:             cluster.ClusterID,
+		CloudProvider:         cluster.CloudProvider,
+		Region:                cluster.Region,
+		MultiAZ:               cluster.MultiAZ,
+		Schedulable:           true,
+		KafkaInstanceLimit:    2,
+		Status:                cluster.Status,
+		SupportedInstanceType: cluster.SupportedInstanceType,
+	})
+	enableManualScaling = true
+
+	err = h.Env.CreateServices()
+	g.Expect(err).ToNot(gomega.HaveOccurred(), "unable to initialize testing environment: %v", err)
+
+	h.Env.Start()
```

Comment:
```suggestion
	// ensures that the data plane is ready
	_, getClusterErr := common.WaitForClusterStatus(h.DBFactory(), &kafkatest.TestServices.ClusterService, clusterID, api.ClusterReady)
	g.Expect(getClusterErr).ToNot(gomega.HaveOccurred(), "failed to get dataplane cluster details: %v", getClusterErr)
```

Reply:
i've done the same here

Reply:
+1. I think registering the cluster manually during startup removed the need to Stop() and Start() the services.